test/autotest.py

```diff
@@ -626,7 +612,7 @@ def testCheckpoint():
           "unexpected number of checkpoint files, %s procs, %d files"
           % (str(status[0]), numFiles))
 
-    if SLOW > 1 and CKPT_CMD != b'Kc':
+    if SLOW > 1 and CKPT_CMD != "Kc":
```

Comment:
Same here. We don't need to break Python 2.7 for no special reason.
'Kc' and "Kc" are equivalent. But removing b is not equivalent.
Please do: b'Kc' -> b"Kc".